frontend/package.json

```diff
@@ -144,9 +144,9 @@
 		"@types/validator": "^13.1.3",
 		"@typescript-eslint/eslint-plugin": "^5.46.1",
 		"@typescript-eslint/parser": "^5.46.1",
-		"@vanilla-extract/css": "^1.11.1",
-		"@vanilla-extract/esbuild-plugin": "^2.2.2",
-		"@vanilla-extract/vite-plugin": "^3.8.2",
+		"@vanilla-extract/css": "^1.13.0",
+		"@vanilla-extract/esbuild-plugin": "^2.3.0",
+		"@vanilla-extract/vite-plugin": "^3.9.0",
```

Comment:
Was attempting to fix HMR when making changes to styles in the UI package and upgraded these packages as part of it. I didn't get it fixed, but figured it couldn't hurt to leave upgrades in.